src/components/tiptap/floatingMenu/floatingMenuSuggestion.tsx

```diff
@@ -85,14 +85,14 @@ export const floatingMenuSuggestion = {
           }
         },
       },
-      {
-        title: 'Embed',
-        command: ({ editor, range }: { editor: Editor; range: any }) => {
-          const tiptapEditorUtils = new TiptapEditorUtils(editor)
-          tiptapEditorUtils.deleteRange(range)
-          tiptapEditorUtils.insertCodeBlock()
-        },
-      },
+      // {
```

Comment:
Lets remove it. We can go back to the old commits if we need it in the future.

Reply:
next commit resolves this @sazanrjb